Benchmark-Models/model_info.py

```diff
@@ -22,6 +24,19 @@
 index_column = 'name'
 
 
+def check_events(petab_problem: petab.Problem) -> int:
+    """Check for events in the model"""
+    n_events = len(petab_problem.sbml_model.getListOfEvents())
+
+    parameter_ids = [p.getId() for p in
+                     petab_problem.sbml_model.getListOfParameters()]
+    for p_id in parameter_ids:
+        assignment_rule = petab_problem.sbml_model.getAssignmentRule(p_id)
+        if assignment_rule and 'piecewise' in assignment_rule.getFormula():
```

Comment:
That will also count any variable including `piecewise` in its name, but probably okay for now.

Reply:
This might work
```python
import sympy as sp
if assignment_rule and any([
    arg.__class__.__name__ == 'piecewise'
    for arg in sp.sympify(assignment_rule.getFormula()).args
]):
    # expression has "piecewise" function
    pass
```